hack/update-codegen.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT_DIR="$(dirname "${BASH_SOURCE[0]}")/.."
+
+CODE_GENERATOR_DIR=$(echo $(mkdir -p "${ROOT_DIR}"/bin; cd "${ROOT_DIR}"/bin; pwd))
+CODE_GENERATOR_VERSION="v0.26.0"
+CONTROLLER_TOOLS_VERSION="v0.11.1"
+
+KWOK_PROJECT="sigs.k8s.io/kwok"
+KWOK_API_PACKAGE="${KWOK_PROJECT}/pkg/apis"
+
+function code_generator() {
+    for gen in `echo {deepcopy-gen,defaulter-gen,conversion-gen,client-gen,lister-gen,informer-gen}`; do
+        if [ ! -f "${CODE_GENERATOR_DIR}/${gen}" ]; then
+            GOBIN="${CODE_GENERATOR_DIR}" go install "k8s.io/code-generator/cmd/${gen}@${CODE_GENERATOR_VERSION}"
+        fi
+    done
+}
+
+function gen() {
```

Comment:
maybe you can directly call https://github.pie.apple.com/kubernetes/code-generator/blob/master/generate-internal-groups.sh to generate all you want.

scheduler-plugins has an [example](https://github.com/kubernetes-sigs/scheduler-plugins/blob/2502825c671063af5b2aa78a1d34b24917f2def4/hack/update-codegen.sh#L36-L42), but it's up to 1.26 yet. FYI.

Reply:
Hi @Huang-Wei, I found scheduler-plugins can also NOT run outside GOPATH, and I created a PR to fix it at the same time.
https://github.com/kubernetes-sigs/scheduler-plugins/pull/481